rules-reviewed/eap7/eap6/hibernate4.windup.xml

```diff
@@ -10,9 +10,10 @@
             <addon id="org.jboss.windup.rules,windup-rules-java,2.4.0.Final" />
         </dependencies>
         <sourceTechnology id="hibernate" versionRange="[4,)" />
```

Comment:
well, it doesn't matter much as we don't have that implemented :(, anyway I will add 5 there